### DISCUSSION TITLE
change power output of the inverter , to cool down the inverter by overheating

### DISCUSSION THREAD

**Discussion by samhac99** [tijsverkoyen/HomeAssistant-FusionSolar#183] [Ideas]
Hello, 
Should it be possible to change the power output of the inverter . (make a identiteits id : number) , so that you can lower de output of the inverter  (e.g. from 10kw to 8kw ) when the  "sensor.inverter_1_inverter_internal_temperature" becomes higher than a certain degree ( e.g. 85°C) to cool down the inverter. , You can do this manually in the fusion app but it should be a win to have this automated.

You lose some power but , your inverter wil last longer

Thanks Sam

**Comment by tijsverkoyen** [comment]
At this point I will not invest time in such a feature. The integration is only reading data and not writing anything back.
If you want to automate this, I think it is possible with https://github.com/wlcrs/huawei_solar
